common/src/bitset.rs

```diff
@@ -302,6 +308,67 @@ impl BitSet {
     pub fn tinyset(&self, bucket: u32) -> TinySet {
         self.tinysets[bucket as usize]
     }
+
+    /// next_set_bit returns the lowest set bit >= `start`, or -1 if none.
```

Comment:
```suggestion
    /// next_set_bit returns the lowest set bit >= `start`, or -1 if none.
```

-1 is not possible. This method returns a u32.